server/list.js

```diff
@@ -171,8 +171,8 @@ function produceTree(files, firstParent) {
 
     // prepare data for the individual file and store later for reference
     // FIXME: consider how to remove circular dependency here.
```

Comment:
```suggestion
```
Since we've addressed the circular deps in this PR, should we remove this comment?